components/filter2/dropdown_group/view.coffee

```diff
@@ -35,7 +35,7 @@ module.exports = class DropdownGroupView extends Backbone.View
 
   toggleForSale: ->
     if @params.get('for_sale') is 'true'
-      @params.set 'for_sale', 'false'
+      @params.set 'for_sale', null
```

Comment:
Yep, tried this. It works for the url on the API, but then the client API isn't updated. So when toggling back and forth the client stays at `?for_sale=true`.